### ISSUE TITLE
Attempt to invoke virtual method 'boolean guichaguri.trackplayer.logic.track.Track.needsNetwork()' on a null object reference

### ISSUE THREAD

**Issue by jasongrishkoff** [Guichaguri/react-native-track-player#22]
Another little crash that's popping up:

```
06-12 13:30:56.934  6738  6994 E AndroidRuntime: Process: com.decoderhq.indieshuffle, PID: 6738
06-12 13:30:56.934  6738  6994 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean guichaguri.trackplayer.logic.track.Track.needsNetwork()' on a null object reference
06-12 13:30:56.934  6738  6994 E AndroidRuntime:        at guichaguri.trackplayer.logic.MediaManager.onPlayerStop(MediaManager.java:317)
06-12 13:30:56.934  6738  6994 E AndroidRuntime:        at guichaguri.trackplayer.logic.MediaManager.onStop(MediaManager.java:169)
06-12 13:30:56.934  6738  6994 E AndroidRuntime:        at guichaguri.trackplayer.player.Player.updateState(Player.java:213)
06-12 13:30:56.934  6738  6994 E AndroidRuntime:        at guichaguri.trackplayer.player.players.ExoPlayer.onPlayerStateChanged(ExoPlayer.java:199)
06-12 13:30:56.934  6738  6994 E AndroidRuntime:        at com.google.android.exoplayer2.ExoPlayerImpl.handleEvent(ExoPlayerImpl.java:326)
06-12 13:30:56.934  6738  6994 E AndroidRuntime:        at com.google.android.exoplayer2.ExoPlayerImpl$1.handleMessage(ExoPlayerImpl.java:93)
06-12 13:30:56.934  6738  6994 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:102)
06-12 13:30:56.934  6738  6994 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:148)
06-12 13:30:56.934  6738  6994 E AndroidRuntime:        at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:208)
06-12 13:30:56.934  6738  6994 E AndroidRuntime:        at java.lang.Thread.run(Thread.java:818)
06-12 13:30:56.938  1618  2967 W ActivityManager:   Force finishing activity com.decoderhq.indieshuffle/.MainActivity

```

**Comment by jasongrishkoff** [comment]
This seems to happen when I start one queue/playlist, then change my mind and start a different playlist.

I was able to work around it by changing line 317 of `logic/MediaManager.java` to:
` if(player.getCurrentTrack() != null && player.getCurrentTrack().needsNetwork()) ...`

**Comment by Guichaguri** [comment]
Fixed in 6bd005255ebdf360cba2f2467d18fec63e4292f0
